src/System.Management.Automation/engine/remoting/commands/PSRemotingCmdlet.cs

```diff
@@ -561,6 +562,7 @@ public virtual PSCredential Credential
         /// to override the policy setting
         /// </remarks>
         [Parameter(ParameterSetName = PSRemotingBaseCmdlet.ComputerNameParameterSet)]
+        [Parameter(ParameterSetName = PSRemotingBaseCmdlet.SSHHostParameterSet)]
```

Comment:
Just tested this, confirmed to default to ComputerNameParameterSet:

PS C:\Users\Lee\Documents\PowerShell> New-PSSession localhost -Port 5985

 Id Name            ComputerName    ComputerType    State         ConfigurationName     Availability
 -- ----            ------------    ------------    -----         -----------------     ------------
 11 WinRM11         localhost       RemoteMachine   Opened        Microsoft.PowerShell     Available

Not sure whether DefaultParameterSetName would help here to remove any uncertainty?